sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFluxBase.java

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.rest;
+
+import org.reactivestreams.Publisher;
+import reactor.core.CoreSubscriber;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+/**
+ * This class is a flux that can operate on a {@link PagedResponse} and
```

Comment:
```suggestion
 * This class is a flux that can operate on any type that extends {@link PagedResponse} and
```
I guess this is the only distinction between PagedFlux and PagedFluxBase.

Reply:
updated.